plugins/inputs/tail/testdata/multiline_quoted_single.csv

```diff
@@ -0,0 +1,9 @@
+1660819827410,1,some text without quotes,A
+1660819827420,2,some text with 'quotes',B
+1660819827430,3,some text with 'multiple "quotes" in `one` line',C
+1660819827440,4,some multiline text with 'quotes
+spanning \'multiple\'
+lines' but do not 'end
+directly',D
+1660819827450,5,all of 'this' should 'basically' work...,E
```

Comment:
Shouldn't the test also include a line with the field completely quoted?
```suggestion
1660819827450,5,all of 'this' should 'basically' work...,E
1660819827460,6,'all of this should
basically work...',F
```

As I guess that will be the most common use case?

Reply:
Agreed. Will add.